### PR TITLE
add ssl certificates schema

### PR DESCRIPTION
**Pull request by malmstein**
**Asana Task/Github Issue:**
https://app.asana.com/0/1142021229838617/1206999322463115

## Description
We can now bypass sites with ssl certificates issues. This PR adds a schema for that, putting the feature behind a flag only enabled for internal users.

#### Reference
- [Config Reviewer Documentation](https://app.asana.com/0/1200890834746050/1204443212791216/f)
- [Config Maintainer Documentation](https://app.asana.com/0/1200890834746050/1200573250322769/f)
- [Feature Implementer Documentation](https://app.asana.com/0/1200890834746050/1201498956177210/f)

### CODE REVIEW

File: overrides/macos-override.json [hunk 1]
```diff
@@ -865,6 +865,15 @@
                     "state": "enabled"
                 }
             }
+        },
```
Comment (marcosholgado) [inline overrides/macos-override.json:868]: Do we want this on macos?

**Comment by marcosholgado** [comment]
Looks like this is added to every single platform, do we want that?

Also, do you want the exceptions to apply automatically too? Gonna guess no, in which case you want to follow https://app.asana.com/0/1200890834746050/1201498956177210/f to remove them.

File: overrides/macos-override.json (hunk 1 above)
Comment (malmstein) [reply overrides/macos-override.json:868]: @SabrinaTardio  asked for it. https://app.asana.com/0/0/1206753149050909/1207007243483505/f

**Comment by malmstein** [comment]
@marcosholgado  I’ve disabled it for all platforms that don’t override and removed the default exception list.

**Comment by malmstein** [comment]
@SlayterDev let me know if there’s anything else

**Pull request merged**